package.json

```diff
@@ -49,6 +49,7 @@
   },
   "devDependencies": {
     "ajv": "^6.10.2",
+    "api-testing": "^1.4.1",
```

Comment:
Ok, last comment I promise. Can you bump this 1.4.2? I found a bug yesterday and released a new version

Reply:
No worries @clarakosi, the comments can keep coming. `api-testing` has been updated from 1.4.1 to 1.4.2.